apps/admin-be/src/types/ProductType.ts

```diff
@@ -0,0 +1,22 @@
+export type ProductSizeTypes =
+  | "3xs"
+  | "xxs"
+  | "xs"
+  | "s"
+  | "m"
+  | "l"
+  | "xl"
+  | "2xl"
+  | "3xl";
+
+export type ProductCategoryType = string;
+
+export type ProductType = {
+  id: string;
+  name: string;
+  price: number; // Price of the product in cents ($19 = 1900).
+  images: string[]; // URL of the images.
+  sizes: ProductSizeTypes[];
+  productCategory: ProductCategoryType;
+  isAvailable?: boolean;
+};
```

Comment:
We should move this to a new lib in `packages/types`
More info here: https://turbo.build/repo/docs/handbook/sharing-code

Reply:
Yep, will move.